packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -356,6 +364,24 @@ export class ComponentPage extends React.Component<IComponentPageProps, {}> {
     return undefined;
   }
 
+  private _getFeedback(): JSX.Element | undefined {
+    if (this.props.feedback) {
+      return (
+        <div className="ComponentPage-feedbackSection">
+          <h2 className="ComponentPage-subHeading ComponentPage-variantsTitle" id="Feedback">
+            Feedback
+          </h2>
+          <a href="https://github.com/OfficeDev/office-ui-fabric-react/issues" target="_blank">
```

Comment:
I'm wondering if this should take users directly to the "Issue template selection" screen vs straight into the issues list? https://github.com/OfficeDev/office-ui-fabric-react/issues/new/choose

![image](https://user-images.githubusercontent.com/307118/42607601-a001d9a0-8538-11e8-8423-afe50d3180ab.png)